### ISSUE TITLE
[Bug]: Failed to download binary for Laravel extension: php-parser-v0.1.17-x64-win32

### ISSUE THREAD

**Issue by edmer664** [laravel/vs-code-extension#86]
### Extension Version

0.1.12

### PHP Binary

Local PHP

### Operating System

Windows

### What happened?

 I encountered an issue where the Laravel VS Code extension fails to download the PHP parser binary. The error message is as follows:
 
`Failed to download binary for Laravel extension: php-parser-v0.1.17-x64-win32`

### Steps to Reproduce:

1. Install the Laravel VS Code extension.
2. Activate the extension.
3. Observe the error message indicating the failure to download the binary.

**Expected Behavior:** The extension should successfully download the PHP parser binary and set it up for use.

**Actual Behavior:** The extension fails to download the binary and displays the error message mentioned above.

**Investigation:** I found that the issue is due to the `downloadBinary` function in the `src/support/parser.ts` file. The generated URI does not exist in the https://github.com/laravel/vs-code-php-parser-cli/releases repository.

### Mimimal Code Sample

```php
File Downloader Log:

`Starting download from https://github.com/laravel/vs-code-php-parser-cli/releases/download/v0.1.17/php-parser-v0.1.17-x64-win32
ERROR: Request failed with status code 404. Technical details: {"message":"Request failed with status code 404","name":"AxiosError","stack":"AxiosError: Request failed with status code 404\n `
```

**Comment by MasterZydra** [comment]
I think there are two changes required to fix that:
1. Updating php-parser version from "0.1.17" to "0.1.31"
2. For windows the file extension ".exe" must be added

**Comment by joetannenbaum** [comment]
We just released v0.1.13 that should have better Windows support, can you verify that this is still an issue after upgrading?

**Comment by edmer664** [comment]
@joetannenbaum the issue regarding the binary download has been fixed. However, there is a new issue where the extension still throws an error, but it does not display any detailed information about what went wrong. 

**Current Behavior:**

- The extension throws an error when trying to open the suggestion pop-up.
- The error message is generic and does not provide any specific details.

Expected Behavior:

- The extension should display detailed error information to help troubleshoot the issue.
- The error message should include specific details about what went wrong.

![image](https://github.com/user-attachments/assets/82c753ad-b4fa-4a68-81d3-344021dac479)

```
2024-12-20 07:51:54.242 [info] Activating Laravel Extension...
2024-12-20 07:51:54.243 [info] Started
2024-12-20 07:52:01.531 [error] 
2024-12-20 07:53:12.224 [error] 
2024-12-20 07:53:15.813 [error] 
2024-12-20 07:53:22.201 [error] 
2024-12-20 07:54:08.172 [info] Stopped
2024-12-20 07:54:26.314 [info] Activating Laravel Extension...
2024-12-20 07:54:26.314 [info] Started
2024-12-20 07:54:32.196 [error] 
```


Will close this issue and will open a new one for this.
